catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -460,10 +461,16 @@ public Response getDocument(@Encoded @PathParam("sourceid") String encodedSource
                 if (transformerParam != null) {
                     transformer = transformerParam;
                 }
-                Filter filter = getFilterBuilder().attribute(Metacard.ID)
+                Filter idFilter = getFilterBuilder().attribute(Metacard.ID)
                         .is()
                         .equalTo()
                         .text(id);
+                Filter tagsFilter = getFilterBuilder().attribute(Metacard.TAGS)
```

Comment:
Why did you need to change the endpoint code?  Shouldn't the catalog framework be taking care of this?